src/main/scala/eastwood/Base.scala

```diff
@@ -0,0 +1,62 @@
+package eastwood
+
+import com.typesafe.config.ConfigFactory
+import io.gatling.core.Predef._
+import io.gatling.http.Predef._
+
+
+class Base extends Simulation {
```

Comment:
Would be good to have some "scaladocs" here

Reply:
Done